Update README.md (#95)
apache/cloudstack-go: README.md

The SDK relies on the  `generate.go` script to auto generate the code for all the supported APIs listed in the `listApis.json` file.
The `listAPIs.json` file holds the output of `listApis` command for a specific release of CloudStack.

> **_NOTE:_** This will only take into account enabled APIs, so only those accessible to the user doing the call at the time of the call. Other users than admin may get less APIs in their list and even admin will not get APIs for services that are not enabled.

Reviewer: ```suggestion
**_NOTE:_** This will only take into account enabled APIs, so only those accessible to the user doing the call at the time of the call. Non-Admin users may get less APIs in their list and even admin will not get APIs for services that are not enabled.
```
